pkg/controllernamespace/namespace/namespace_controller.go

```diff
@@ -75,12 +69,13 @@ func (r *ReconcileNamespace) Reconcile(request reconcile.Request) (reconcile.Res
 		// - customer-admin
 		// - customer-project-admin
 		// - customer-view
-		log.Printf("Reconciling Namespace %s/%s\n", request.Namespace, request.Name)
+		log.Printf("Reconciling Namespace %s/%s", request.Namespace, request.Name)
```

Comment:
move to logrus as all our code?

Reply:
Yeah, I think during my rebase-rework I didn't include these.  Thanks, will fix.